ontology/catplus_ontology.ttl

```diff
@@ -626,7 +626,6 @@ cat:reactionName a rdf:Property ;
 cat:AddAction a rdfs:Class, sh:NodeShape ;
    rdfs:subClassOf allo-real:AFRE_0000001 ;
    sh:property [sh:path cat:dispenseType ; sh:datatype xsd:string] ;
-   sh:property [sh:path cat:hasWell ; sh:class cat:Well ] ;

```

Comment:
Every action has either a `cat:hasPlate` or `cat:hasWell`. Therefore I have removed this property here and added it to the general Action instead. (Bravo Actions also relate to Well meaning certain positions on the plate rather then to the full Plate)